processors/blurc/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# must have TinyGo installed to obtain the TINYGOROOT, or else separately install wasi-lib
+WASI_LIBC_SYSROOT=$(tinygo env TINYGOROOT)
+
+/opt/wasi-sdk/bin/clang --target=wasm32-unknown-unknown -O3 \
+        --sysroot="${WASI_LIBC_SYSROOT}/lib/wasi-libc/sysroot" \
+        -z stack-size=4096 -Wl,--initial-memory=65536 \
+        -o ../blurc.wasm process.c ../../../wasmcv/components/c/wasmcv/imports.c ../../../wasmcv/components/c/wasmcv/imports_component_type.o \
```

Comment:
This path probably needs to be set by an env var or something, so it can match the directory to which the user has installed the wasmcv C files.